fe1-web/store/reducers/.eslintrc

```diff
@@ -2,6 +2,7 @@
   "root": false,
   "rules": {
     /* All reducers use the createSlice function, which requires the user to param-reassign */
-    "no-param-reassign": "off"
+    "no-param-reassign": "off",
+    "import/no-named-as-default": 0
```

Comment:
Otherwise, the CI fails because of `Using exported name 'socialReducer' as identifier for default export  import/no-named-as-default`, I found this solution here https://stackoverflow.com/questions/44437203/how-do-i-resolve-eslint-import-no-named-as-default

Reply:
Ah no, actually this is because of export socialReducer, I will remove both of them.

Reply:
Things fix with the socialReduce, I will remove it